core/test/acceptance/old/admin/themes_spec.js

```diff
@@ -22,17 +22,13 @@ describe('Themes API', function () {
             .set('Origin', config.get('url'))
             .attach(fieldName, themePath);
     };
-
     before(function () {
-        return ghost()
```

Comment:
`mocha` uses the return value (if it's a promise) in `before`, `beforeEach` etc.. to know when the setup/teardown has completed. So remove of the `return` here is changing functionality. I think you'll want to either update these to `async` functions and `await` the operations, continue to `return` promises OR change it to `before(function (done) {` and call `done` when everything is setup!